### ISSUE TITLE
Did the non-ERCF files get overwritten?

### ISSUE THREAD

**Issue by EllaFoxo** [Eytecz/LGX_Lite_Stealthburner_CW2_style_mount#16]
Looking back at the commit history, I think they got yanked out in a force-overwrite merge. Was this intentional? I'm looking to print the non-ERCF version, and was wondering if that was still supported

**Comment by hottubwithchampagne** [comment]
Yes, now I have my LGX Lite and looks like there's something missing...
